spec/requests/linking/link_defendant_no_maat_id_spec.rb

```diff
@@ -50,9 +50,8 @@
         expect(flash.now[:alert]).to match(/Defendant is not a valid uuid/i)
       end
 
-      it 'redirects to defendant path' do
-        expect(response).to redirect_to new_laa_reference_path('69a73434-ae4b-4728-97b8-6a0c60f37930',
-                                                               urn: urn)
+      it 'renders laa_reference_path and does not redirect' do
```

Comment:
https://github.com/ministryofjustice/laa-court-data-ui/pull/241/files#r466394574

Reply:
https://github.com/ministryofjustice/laa-court-data-ui/pull/241/files#r466397250

Reply:
👍 removed and does not redirect
Used expect(response).to render_template('new')